conda/meta.yaml

```diff
@@ -43,3 +44,4 @@ requirements:
     - setuptools_scm
     - ts-tcpip
     - ts-utils
+    - pytest
```

Comment:
Are you sure you want to add `pytest` to the "run" dependency? Do you really need it to run the package? Consider making the module that depends on `pytest` a submodule that is not imported by default. I am really weary of requiring "test" packages as a run dependency.

Reply:
Good point, thanks! Done.